docs/general/administration/hardware-selection.md

```diff
@@ -55,7 +55,7 @@ For low power applications, Intel 12th gen or newer Atom CPUs with integrated gr
 
 :::caution SBCs (Single Board Computers)
 
-Most SBCs use low powered chipsets, often with less than ideal driver support from the chipset vendors. They are generally too slow for a good experience and/or have broken hardware acceleration support. Please avoid using SBCs such as Raspberry Pis (Including newer Raspberry Pi 5 Models) to run Jellyfin.
+Most SBCs use low powered chipsets, often with less than ideal driver support from the chipset vendors. They are generally too slow for a good experience and/or have broken hardware acceleration support. Please avoid using SBCs such as Raspberry Pis (Including newer Raspberry Pi 5 Models) to run Jellyfin. **One exception** is that the **Rockchip RK3588/3588S SoC based SBCs** provide excellent transcoding speed and power efficiency - H.264/HEVC up to 1080p@480fps or 4k@120fps transcoding with **single-digit** power consumption in watts. Another bonus is that it has **10-bit H.264 (High10) and AV1 decoding** support.
```

Comment:
please disclaim that RK3588 is only supported in 10.9 or later

Reply:
I was told that the RK3588 doesn't have good software support on Linux. If that is the case then the wording should be clearer